packages/client/hmi-client/src/composables/activeProject.ts

```diff
@@ -1,5 +1,5 @@
-import { computed, shallowRef } from 'vue';
+import { computed, ref } from 'vue';
 import type { Project } from '@/types/Types';
 
-export const activeProject = shallowRef<Project | null>(null);
+export const activeProject = ref<Project | null>(null);
```

Comment:
Any particular reason we need a ref instead of shallowRef?

Reply:
Yes, that was part of a different PR that I'm working on. I can revert that before merging for sure.